tests/yam/agama/boot_home_encryption.pm

```diff
@@ -0,0 +1,41 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Boot the system for home encryption scenario.
+# - Select first entry to boot in grub2
+# - Enter passphrase for home partition.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Backends;
+
+sub run {
+    my $enter_passphrase_for_home = $testapi::distri->get_enter_passphrase_for_home();
+    my $grub_menu = $testapi::distri->get_grub_menu_installed_system();
+
+    if (is_svirt) {
+        select_console('svirt');
+
+        # enter passphrase for home encryption
+        wait_serial("Please enter passphrase for disk cr_home.*", 300);
```

Comment:
new `boot_encrypted_partitions.pm` should be able to handle this, without creating new file.

Reply:
removed this module.